e2e/playwright/desktop-export.spec.ts

```diff
@@ -112,60 +104,53 @@ test(
       const u = await getUtils(page)
       await u.openFilePanel()
 
+      // Click on the other file
       const otherKclButton = page.getByRole('button', { name: 'other.kcl' })
-
-      // Click the file
       await otherKclButton.click()
 
       // Close the file pane
       await u.closeFilePanel()
+      await scene.waitForExecutionDone()
+      await page.waitForTimeout(1_000) // wait for panel buttons to be available
 
-      // FIXME: await scene.waitForExecutionDone() does not work. The modeling indicator stays in -receive-reliable and not execution done
-      await page.waitForTimeout(10000)
-
-      // expect zero errors in guter
+      // Expect zero errors in gutter
       await expect(page.locator('.cm-lint-marker-error')).not.toBeVisible()
 
-      // export the model
+      // Click the export button
       const exportButton = page.getByTestId('export-pane-button')
       await expect(exportButton).toBeVisible()
-
-      const gltfOption = page.getByText('glTF')
-      const submitButton = page.getByText('Confirm Export')
-      const exportingToastMessage = page.getByText(`Exporting...`)
-      const errorToastMessage = page.getByText(`Error while exporting`)
-      const engineErrorToastMessage = page.getByText(`Nothing to export`)
-      const alreadyExportingToastMessage = page.getByText(`Already exporting`)
-      // The open file's name is `other.kcl`, so the export file name should be `other.gltf`
-      const exportFileName = `other.gltf`
-
-      // Click the export button
       await exportButton.click()
+      await page.waitForTimeout(1_000) // wait for export options to be available
```

Comment:
That's a little sus no? Considering that the options are all hardcoded there https://github.com/KittyCAD/modeling-app/blob/bfdf8babed009ee6c8c208dd2d5d9dd5718c77a7/src/lib/commandBarConfigs/modelingCommandConfig.ts#L236-L242

Reply:
I think I'd prefer to see a cmdBar.expectState call here. It's probably because this test is very old that it wasn't there